src/blocks/scratch3_countdown.js

```diff
@@ -125,39 +24,12 @@ class Scratch3Countdown {
             messages_deleteSubscriptions: this.deleteSubscriptions,
             messages_assignTopicToMessage: this.assignTopicToMessage,
             messages_unassignTopicToMessage: this.unassignTopicToMessage,
-            countdown_gameMode: this.gameMode,
             countdown_startCelebration: this.startCelebration,
             countdown_whenTimerStarted: this.whenTimerStarted,
-            countdown_whenCelebrationStarted: this.whenCelebrationStarted,
-            countdown_gameModeCheck: this.gameModeCheck
-        };
-    }
-
-    getHats () {
-        return {
-            event_whenstarted: {
-                restartExistingThreads: true
-            }
+            countdown_whenCelebrationStarted: this.whenCelebrationStarted
```

Comment:
Are we even using the Countdown class anymore?

Reply:
no we are not, good catch. will removed commented & associated code as well in the next PR